Src/WitsmlExplorer.Frontend/contexts/filter.tsx

```diff
@@ -255,6 +230,21 @@ const filterOnName = (wells: Well[], filter: Filter, filterOptions: FilterOption
   return filteredWells;
 };
 
+export const getSearchRegex = (str: string, findEmpty: boolean): RegExp => {
+  let newStr = str;
+  if (findEmpty) {
+    newStr = "^$"; // Empty string
+  } else if (str == "") {
```

Comment:
The expression `str == ""` will return false if `str` only contains whitespaces. Is there any need to disallow a string containing only whitespaces?

Reply:
I will allow searching for whitespaces for now. This can be changed in the future if people get confused.